website_event_snippet_calendar/templates/snippets.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Tecnativa - Jairo Llopis
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <!-- Individual snippets -->
+    <template id="loading" name="Loading">
+        <div>
+            <i class="fa fa-spinner fa-spin"/> Loading...
+        </div>
+    </template>
+
+    <template id="s_event_calendar" name="Events Calendar">
+        <div class="o_not_editable css_non_editable_mode_hidden alert alert-info">
+            <p>Calendar visible only when you save</p>
+        </div>
+        <div t-ignore="true"
+             class="s_event_calendar o_not_editable css_editable_mode_hidden">
+            <t t-call="website_event_snippet_calendar.loading"/>
+        </div>
+    </template>
+
+    <template id="s_event_list" name="Events List">
+        <div class="o_not_editable css_non_editable_mode_hidden alert alert-info">
+            <p>Event list visible only when you save.</p>
+            <p>It will display up to the closest <b class="js_amount">4</b> events.</p>
+        </div>
+        <div t-ignore="true"
+             class="s_event_list o_not_editable css_editable_mode_hidden">
+            <t t-call="website_event_snippet_calendar.loading"/>
+        </div>
+    </template>
+
+    <template id="s_event_calendar_list" name="Events Calendar and List">
+        <section class="oe_snippet_body s_event_calendar_list">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12 mt32 text-center">
+                        <h3>Events</h3>
+                    </div>
+                    <div class="col-md-4 mt16 mb16">
+                        <h4>Calendar</h4>
+                        <t t-call="website_event_snippet_calendar.s_event_calendar"/>
+                    </div>
+                    <div class="col-md-8 mt16 mb16">
+                        <h4>List</h4>
+                        <t t-call="website_event_snippet_calendar.s_event_list"/>
+                    </div>
+                    <div class="col-md-12 mb32 text-right">
+                        <a href="/events" class="btn btn-info">All events</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </template>
+
+    <!-- Add snippets to menu -->
+    <template id="snippets" inherit_id="website.snippets">
+        <xpath expr="//*[@id='snippet_feature']//*[hasclass('o_panel_body')]">
+            <t t-snippet="website_event_snippet_calendar.s_event_calendar_list"
+               t-thumbnail="/website_event_snippet_calendar/static/description/icon.png"/>
+        </xpath>
+    </template>
+
+    <!-- Add snippets options -->
+    <template id="snippet_options" inherit_id="website.snippet_options">
+        <xpath expr=".">
+            <!-- Allow to add or remove events from the list -->
```

Comment:
Please, document in README :wink:

Reply:
Done in https://github.com/OCA/event/pull/101/commits/3b6395983ad82d86d8ffe98950de222873190d39